nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/GFFileHandler.java

```diff
@@ -585,14 +573,10 @@ private void configureUniformLogFormatter(String excludeFields, boolean multiLin
     }
 
     void initializePump() {
-        if (pumpFuture != null) {
-            pumpFuture.cancel(true);
-            pumpFuture = null;
-        }
-        if (logToFile) {
-            pumpFuture = payaraExecutorService.submit(
-                () -> {
-                    while (!done.isSignalled() && logToFile) {
+        pump = new Thread() {
```

Comment:
It might be worth putting in a comment explaining why we're not using a the managed executor service here so that we don't come back to this in a year's time and go "why is that not using a managed executor?"